Tests/test_ProtParam.py

```diff
@@ -139,11 +139,11 @@ def test_flexibility(self):
     def test_isoelectric_point(self):
         """Calculate the isoelectric point."""
         # Old test used a number rounded to two digits, so use the same
-        self.assertAlmostEqual(self.analysis.isoelectric_point(), 7.72, 2)
+        self.assertAlmostEqual(self.analysis.isoelectric_point(), 6.96, 2)
```

Comment:
Does this mean that the default behavior changed? This would be bad for backward compatibility.
Was there an error in the calculation before or do you set a different method as default?

Reply:
Yes, the default method changed (previously Bjellqvist pKa sets inside of the algorithm have been used, now it is IPC2_protein set). The reason for the change is that IPC2_protein gives better results (http://ipc2.mimuw.edu.pl/results.html) in comparison to Bjellqvist set introduced in 1994.